x-pack/test/api_integration/apis/management/rollup/rollup.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { API_BASE_PATH, ROLLUP_INDEX_NAME } from './constants';
+
+import { registerHelpers } from './rollup.test_helpers';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  const {
+    createIndexWithMappings,
+    getJobPayload,
+    createJob,
+    deleteJob,
+    startJob,
+    stopJob,
+    cleanUp,
+  } = registerHelpers({ supertest, es });
+
+  describe('Rollup jobs', () => {
+    after(() => cleanUp());
+
+    describe('Rollup indices', () => {
+      it('should return an empty object when there are no rollup indices', async () => {
+        const uri = `${API_BASE_PATH}/indices`;
+
+        const { body } = await supertest
+          .get(uri)
+          .expect(200);
+
+        expect(body).to.eql({});
+      });
+    });
+
+    describe('Index patterns', () => {
+      it('should return the date, numeric and keyword fields when an index (pattern) is found', async () => {
```

Comment:
How about `when an index pattern matches indices`? This would be a bit more accurate and reflect the language in our UI.